Update docs-installation.mdx

JustAnotherDevv opened the pull request:
- Fixed a typo in page slug
- Checked that content is up-to-date and removed LegacyContentBanner

JustAnotherDevv committed:
Update docs-installation.mdx

jthiller commented on docs/home/faq/docs-installation.mdx:
```diff
@@ -2,18 +2,14 @@
 description: Docs Installation Guide
 image: https://docs.helium.com/img/link-image.png
-slug: /home/faqdocs-installation
+slug: /home/faq/docs-installation
```
The build is failing due to the url change.

JustAnotherDevv committed:
Update docs-installation.mdx

JustAnotherDevv committed:
Update docs-installation.mdx

JustAnotherDevv committed:
Update docs-installation.mdx

JustAnotherDevv committed:
Update docs-installation.mdx

JustAnotherDevv committed:
Update docs-installation.mdx

JustAnotherDevv committed:
Update open-source.mdx

JustAnotherDevv commented:
@jthiller My bad, the url slug was used somewhere else too and that caused the build error. I've fixed this already.

JustAnotherDevv committed:
Update docs-installation.mdx

Changed `Contributing Documentation`  to `Documentation Contributing` as it seems more proper

jthiller committed:
Update Sidebar Label
